.github/actions/to-integration-tests/dist/to-integration-test.js

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
Did you mean to commit this file?

Reply:
Yes. TypeScript actions typically commit the transpiled JS file, the action itself is not responsible for transpiling.

Reply:
This file is named `to-integration-test.js` there is another in dist called `to-integration-tests.js` which corresponds to the `to-integration-tests.ts` file.

Reply:
oh

Reply:
Removed in 9bee219470